src/app.d.ts

```diff
@@ -2,8 +2,11 @@
 // for information about these interfaces
 // and what to do when importing types
 declare namespace App {
-	// interface Locals {}
-	// interface PageData {}
-	// interface Error {}
-	// interface Platform {}
+	interface Supabase {
+		Database: import('./DatabaseDefinitions').Database;
```

Comment:
I don't know what this import is. I don't see it anywhere.

Also, i'm not sure these Supabase typings should live in this file.

Generally, this `app.d.ts` file should rather be for SvelteKit specific type definitions.